### PR TITLE
Droid changes for Factory rework

### PR DESCRIPTION
**Pull request by fritsch**
Platform maintainer decided to register in XBMCApp.

### CODE REVIEW

File: xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp [hunk 1]
```diff
@@ -26,6 +26,8 @@
 #include <androidjni/Build.h>
 
 #include "cores/AudioEngine/Utils/AEUtil.h"
+#include "cores/AudioEngine/AESinkFactory.h"
+#include "cores/AudioEngine/Engines/ActiveAE/ActiveAE.h"
```
Comment (FernetMenta) [inline xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp]: are you sure?

Comment (fritsch) [reply xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp]: To be honest: no. Normally it has nothing to do there. But without it build could not resolve AE namespace.

**Comment by fritsch** [comment]
build pending: http://jenkins.kodi.tv/job/Android-ARM/7656/ <- let's see if the error comes back.

**Comment by fritsch** [comment]
Thanks very much. Killed the same from AESinkPULSE - I don't remember why it was in there. It was obviously wrong having a circular dependency between AE'Sink and Sink and AE.

Thanks much.

**Pull request merged**